client/flutter/lib/components/recent_numbers_graph.dart

```diff
@@ -33,8 +34,26 @@ class _RecentNumbersBarGraphState extends State<RecentNumbersBarGraph> {
 
     return BarChart(
       BarChartData(
-        axisTitleData: FlAxisTitleData(show: false),
-        borderData: FlBorderData(show: false),
+        axisTitleData: FlAxisTitleData(
+          show: false,
+        ),
+        borderData: FlBorderData(
+          show: false,
+        ),
+        barTouchData: BarTouchData(
+          touchTooltipData: BarTouchTooltipData(
+              getTooltipItem: (barChartGroupData, _, barRodData, index) {
+            final date = DateTime.fromMillisecondsSinceEpoch(
+                widget.timeseries[index].epochMsec.toInt());
+            final formattedDate = DateFormat('M/d/y').format(date);
```

Comment:
this format is wrong for most of the non-US world

Reply:
@theswerd - see this earlier comment from me about using "October 6, 2020" as the format to avoid localization issues: https://github.com/WorldHealthOrganization/app/issues/1575#issuecomment-706651820

Reply:
as long as you force en_US somehow as the locale you should be fine, but the date for the month names has to be loaded in dynamically otherwise.  See https://pub.dev/documentation/intl/latest/intl/DateFormat-class.html

Reply:
to be more specific, i think disabling the localizations forces the language part of the tag to be 'en' but I am unclear whether v1 would launch with 'en_NG' or 'en_US' unless someone has tested this system in nigeria yet

Reply:
We're reaching out to the NCDC. They'll be involved with the next round of testing to check out things like this.